crates/iota/src/client_commands.rs

```diff
@@ -3069,29 +3069,29 @@ fn opts_from_cli(opts: HashSet<EmitOption>) -> IotaTransactionBlockResponseOptio
             .with_balance_changes()
     } else {
         IotaTransactionBlockResponseOptions {
-            show_input: opts.contains(&EmitOption::Input),
+            show_input: opts.contains(&DisplayOption::Input),
             show_raw_input: false,
             show_effects: true,
             show_raw_effects: false,
-            show_events: opts.contains(&EmitOption::Events),
-            show_object_changes: opts.contains(&EmitOption::ObjectChanges),
-            show_balance_changes: opts.contains(&EmitOption::BalanceChanges),
+            show_events: opts.contains(&DisplayOption::Events),
+            show_object_changes: opts.contains(&DisplayOption::ObjectChanges),
+            show_balance_changes: opts.contains(&DisplayOption::BalanceChanges),
         }
     }
 }
 
-pub(crate) fn parse_emit_option(s: &str) -> Result<HashSet<EmitOption>, String> {
+pub(crate) fn parse_display_option(s: &str) -> Result<HashSet<DisplayOption>, String> {
     let mut options = HashSet::new();
 
     // Split the input string by commas and try to parse each part
     for part in s.split(',') {
         let part = part.trim(); // Trim whitespace
-        match EmitOption::from_str(part) {
+        match DisplayOption::from_str(part) {
             Ok(option) => {
                 options.insert(option);
             }
-            Err(_) => return Err(format!("Invalid emit option: {}", part)), /* Return error if
-                                                                             * invalid */
+            Err(_) => return Err(format!("Invalid display option: {}", part)), /* Return error if
```

Comment:
use `bail!` ?

Reply:
Yes separate, there's maybe more of this, open an issue if you want

Reply:
https://github.com/iotaledger/iota/issues/6475